runtime/ftplugin/sml.vim

```diff
@@ -0,0 +1,29 @@
+" Vim filetype plugin file
+" Language: 		SML
+" Filenames:		*.sml *.sig
+" Maintainer: 		tocariimaa <tocariimaa@firemail.cc>
+" Last Change:		2025-10-03
+
+if exists('b:did_ftplugin')
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
those are only needed, if you are using line-continuations (or intend to do so in the future).